### ISSUE TITLE
Braces indentation with more than one list item

### ISSUE THREAD

**Issue by zweiter** [perltidy/perltidy#54]
I think there's a discrepancy in the indentation of braces when there's a list with (a trailing comma) or more than one items.
```pl
my $definition =
    (
    {
       key => $value
    },
    ) ;

my $definition =
    (
    {
       key => $value
    },
    {
       key => $value
    }
    ) ;
```
That's what it's doing (and what's intended) when there's only one entry (and no trailing comma):
```pl
my $definition = (
                    {
                      key => $value
                    }
                  ) ;
```

Note: We're using both `--break-before-hash-brace=3` and `--break-before-hash-brace-and-indent=2` together with `-lp`.

**Comment by shancock9** [comment]
Thanks, I'll look at it

**Comment by zweiter** [comment]
I just wanted to add that we also used `--break-before-paren-and-indent=2`.

**Comment by shancock9** [comment]
Version 20210402 has been released.  It has been updated to fix issues git#54 and git #55.
